packages/web-app-cast/tests/e2e/castFileAction.spec.ts

```diff
@@ -0,0 +1,26 @@
+import { test, Page, expect } from '@playwright/test'
+import { FilesAppBar } from '../../../../support/pages/filesAppBarActions'
+import { loginAsUser, logout } from '../../../../support/helpers/authHelper'
+import { FilesPage } from '../../../../support/pages/filesPage'
+
+let adminPage: Page
+
+test.beforeEach(async ({ browser }) => {
+  const admin = await loginAsUser(browser, 'admin', 'admin')
+  adminPage = admin.page
+})
+
+test.afterEach(async () => {
+  const filesPage = new FilesPage(adminPage)
+  await filesPage.deleteAllFromPersonal()
+  await logout(adminPage)
+})
+
+test.fail('check cast file-action', async () => {
```

Comment:
the extension doesn't work on firefox, webkit and headless chromium
See https://github.com/owncloud/web-extensions/issues/187